packages/zoe/src/contractSupport/percentMath.js

```diff
@@ -0,0 +1,62 @@
+import './types';
+import { assert } from '@agoric/assert';
+import { natSafeMath } from './index';
+
+const { multiply, floorDivide } = natSafeMath;
+
+// make a percentMath object, which represents a unitless fraction between 0
+// and 1. It can be multiplied by ERTP amounts to take a percentage. inverse()
+// produces a new percent object that is the  original's additive complement.
+//
+// The default base for the percent is 100, but higher precision can be
+// specified.
+/** @type {MakePercent} */
+function makePercent(value, base = 100) {
+  assert(
+    value >= 0 && value <= base,
+    `percentages (${value}) must be between 0 and base (${base})`,
+  );
+
+  return harden({
+    scale: (amountMath, amount) => {
+      assert(
+        amountMath.getBrand() === amount.brand,
+        `amountMath must have the same brand as amount`,
+      );
+      return amountMath.make(floorDivide(multiply(amount.value, value), base));
+    },
+    inverse: _ => makePercent(base - value, base),
```

Comment:
I have a hard time with `inverse` as the name for this. In an arithmetic context `inverse` is always used unqualified as the multiplicative inverse. The inverse of 20% would be 5 or 500%. Is there another natural term? Please something much shorter than `arithmeticComplement` ;)

Reply:
'complement' seems to be a thing for a  percent.